packages/web-components/src/listbox/define.ts

```diff
@@ -0,0 +1,4 @@
+import { FluentDesignSystem } from '../fluent-design-system.js';
```

Comment:
We should add a story for a stand along listbox as I see this being used outside of combobox

Reply:
Listbox stories are present under `Components/Dropdown/Listbox` in Storybook. The listbox itself doesn't do much, since the Dropdown handles keyboard navigation via `aria-activedescendant`. This means focus is only ever on the input/button, and the listbox/options never receive real focus. There are ways for the listbox to determine if it's standalone, and it should be straightforward to include keyboard events for those scenarios at a later date.